src/components-examples/material/dialog/dialog-harness/dialog-harness-example.ts

```diff
@@ -0,0 +1,19 @@
+import {Component, TemplateRef, ViewChild} from '@angular/core';
+import {MatDialog, MatDialogConfig} from '@angular/material/dialog';
+
+/**
+ * @title Testing with MatDialogHarness
+ */
+@Component({
+  selector: 'dialog-harness-example',
+  templateUrl: 'dialog-harness-example.html',
+})
+export class DialogHarnessExample {
+  @ViewChild(TemplateRef) dialogTmpl: TemplateRef<any>;
```

Comment:
nit: it's a bit more cognitive work to read out abbreviated variable names. Usually easier to just go with `dialogTemplate` or `dialogTemplateRef`